set-up.R

```diff
@@ -53,33 +53,17 @@ remove_style = function(x){
 
 # function to add model_output_table class to all tables
 add_table_class <- function(x){
-  
   x <- gsub("<table>", "<table class='model_output_table'>", x)
   x
 }
 
-save_formats <- function(to_save, name = F, csv = F){
+save_formats <- function(to_save, name = F){
   if (name == F){
     name <- substitute(to_save)
   }
   saveRDS(to_save, file.path(pct_data, region, paste0(name, ".Rds")))
-  
-  # Simplify data checked with before and after using:
-  # plot(l$gendereq_sideath_webtag)
-  to_save@data <- round_df(to_save@data, 5)
-  
-  # Simplify geom
-  geojson_write( ms_simplify(to_save, keep = 0.1), file = file.path(pct_data, region, name))
```

Comment:
I don't think it will put lots of load, just lots of bandwidth.  Downloading is far less memory intensive than what shiny does normally and as we agreed to merge this https://github.com/npct/pct-shiny/pull/393#issuecomment-247589905 then there is no point generating them.